README.md

```diff
@@ -21,7 +16,114 @@ detect components (the concept of component is taken from Odo and its definition
 
 ## Usage
 
-- [Go library & CLI](go/README.md#Usage)
+### CLI
+
+The Go CLI can be built with the below command:
+
+```bash
+# inside the go/ dir
+$ go build alizer.go
+```
+
+```shell
+$ ./alizer analyze <path>
+$ ./alizer component <path>
+$ ./alizer devfile <path>
+```
+
+### CLI Arguments
+
+#### --log
+
+Sets the logging level of the CLI. The arg accepts only 3 values [`debug`, `info`, `warning`]. The default value is `warning` and the logging level is `ErrorLevel`. Example usage
+
+```shell
+$ ./alizer analyze --log debug <path>
+```
+
+### Library Package
```

Comment:
Maybe a link here to name_detection.md , port_detection.md (or in the component detection section)?

Reply:
added to the component detection section